packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -260,9 +279,20 @@ class Draggable<T extends Object> extends StatefulWidget {
   /// is transformed compared to the child.
   final Offset feedbackOffset;
 
-  /// Where this widget should be anchored during a drag.
+  /// Where this widget should be anchored during a drag. Overridable by
+  /// providing a strategy following the [DragAnchoStrategy] interface like the
+  /// built-in [childDragAnchorStrategy] and [pointerDragAnchorStrategy] or a
+  /// custom implementation.
+  @Deprecated('Use dragAnchorStrategy instead')
```

Comment:
this does not have the effect that i see it as deprecated in android studio. seems like this is not the right place. @chunhtai can you give me a hint?

Reply:
you need to add this in line 210, too
```dart
this.dragAnchor = DragAnchor.child,
```
Also the format is not correct. You can checkout other example
https://github.com/flutter/flutter/blob/fe2708c35bfccd821c62036e1daf79283c0bf2af/packages/flutter/lib/src/cupertino/text_field.dart#L266

Reply:
added it to the other place and followed the style documented in the tree hygiene doc.